### PR TITLE
Add get_quad_ids and get_features tests

### PR DESCRIPTION
**Pull request by bfhealy**
This PR adds new tests for functions in `get_quad_ids.py` and `get_features.py`. The broader intent is to link these up with future tests for `inference.py` and the active learning code in `scope.py`.

### CODE REVIEW

**Comment by bfhealy** [comment]
Note to self to add `os.environ.get("KOWALSKI_TOKEN")` and such to `get_features` so the test passes.

File: scope.py [hunk 1]
```diff
@@ -1439,6 +1439,37 @@ def test(self):
         )
         os.remove(path_gen_features)
 
+        print("Testing get_cone_ids...")
+        id_df = get_quad_ids.get_cone_ids(
+            obj_id_list=['obj1', 'obj2', 'obj3'],
+            ra_list=[40.0, 41.0, 42.0],
+            dec_list=[50.0, 51.0, 52.0],
+        )
+        print(id_df)
```
Comment (mcoughlin) [inline scope.py:1448]: Needed?

**Review by mcoughlin — APPROVED** [review]
Can presumably delete the print. Otherwise LGTM

File: scope.py (hunk 1 above)
Comment (bfhealy) [reply scope.py]: Deleted, thanks!

**Pull request merged**